db/write_thread.cc

```diff
@@ -485,9 +487,9 @@ void WriteThread::EnterAsMemTableWriter(Writer* leader,
   // Allow the group to grow up to a maximum size, but if the
   // original write is small, limit the growth so we do not slow
   // down the small write too much.
-  size_t max_size = 1 << 20;
-  if (size <= (128 << 10)) {
-    max_size = size + (128 << 10);
+  size_t max_size = max_write_batch_group_size_bytes;
+  if (size <= 0.125 * max_write_batch_group_size_bytes) {
+    max_size = size + 0.125 * max_write_batch_group_size_bytes;
```

Comment:
Maybe define a variable with value 0.125 * max_write_batch_group_size_bytes, rather than repeat it twice.

Reply:
Done in latest patch.